src/components/Queue.vue

```diff
@@ -12,7 +12,7 @@
                 {{ message }}
             </div>
             <div v-else class="message">
-                You have <strong>{{ vouchers.length }}</strong> vouchers in your queue.
+                [xXx]You have <strong>{{ vouchers.length }}</strong> vouchers in your queue. These voucher codes will be checked when you submit your queue.
```

Comment:
I think for this card, either I should add a line of code in with a condition in to add the s - or make a separate card about pluralisation because there are a few other places in the app that would need to be fixed at the same time if we're going to use a package. What do you think?